cmake/FindDraco.cmake

```diff
@@ -0,0 +1,92 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
+# based on https://github.com/hlrs-vis/covise/blob/b909e6fad418e23e48726ab44de1bb36e82a24f8/cmake/FindDRACO.cmake#L4
+
+# - Find DRACO
+# Find the DRACO includes and library
+#
+#  DRACO_INCLUDE_DIR - Where to find DRACO includes
+#  DRACO_FOUND       - True if DRACO was found
+
+IF(DRACO_INCLUDE_DIR)
+  SET(DRACO_FIND_QUIETLY TRUE)
+ENDIF(DRACO_INCLUDE_DIR)
+
+FIND_PATH(DRACO_INCLUDE_DIR "draco/draco_features.h"
+  PATHS
+  $ENV{DRACO_HOME}/include
+  $ENV{EXTERNLIBS}/DRACO/include
+  ~/Library/Frameworks
+  /Library/Frameworks
+  /usr/local
+  /usr
+  /sw # Fink
+  /opt/local # DarwinPorts
+  /opt/csw # Blastwave
+  /opt
+  DOC "DRACO - Headers"
+)
+
+SET(DRACO_NAMES draco)
+SET(DRACO_DBG_NAMES dracod)
+
+FIND_LIBRARY(DRACO_LIBRARY NAMES ${DRACO_NAMES}
+  PATHS
+  $ENV{DRACO_HOME}
+  $ENV{EXTERNLIBS}/DRACO
+  ~/Library/Frameworks
+  /Library/Frameworks
+  /usr/local
+  /usr
+  /sw
+  /opt/local
+  /opt/csw
+  /opt
+  PATH_SUFFIXES lib lib64
+  DOC "DRACO - Library"
+)
+
+INCLUDE(FindPackageHandleStandardArgs)
+
+IF(MSVC)
+  # VisualStudio needs a debug version
```

Comment:
Why?  Seems to build fine with the release builds.

Reply:
Sorry -- missed this comment. I'm happy to remove, that was all just inherited from the original version of this file.